public/src/components/app.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow, configure } from 'enzyme';
+import { findByTestAttr } from '../../../test/utils';
+import App from './App';
+
+configure({ adapter: new Adapter() });
```

Comment:
Since you have this in every file you could just set up a setupTests.js file where you have the configuration for Enzyme, so you don't repeat code. https://enzymejs.github.io/enzyme/docs/installation/